flink-formats/flink-csv/src/main/java/org/apache/flink/formats/csv/CsvFormatOptions.java

```diff
@@ -87,5 +87,11 @@ public class CsvFormatOptions {
                             "Optional null literal string that is interpreted as a\n"
                                     + "null value (disabled by default)");
 
+    public static final ConfigOption<Boolean> WRITE_BIGDECIMAL_AS_PLAIN =
+            ConfigOptions.key("write-bigdecimal-as-plain")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription("Optional whether write bigdecimal using scientific notation");
```

Comment:
This needs a better comment since the relationship between the flag values and the scientific notation is not immediately clear to the user of the API. The comment has to explain what is the outcome if this flag is set to true.

Reply:
Thank you for your advice,I've made some updates